src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: #{$card-width};
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__subtitle {
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &__rating {
+    width: 100%;
+    margin-bottom: 24px;
+
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-title {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &-value {
+
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: #060b35;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    background: #00acdc;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+
+    transition: 0.5s;
+  }
+
+  &:hover .card__button {
+    color: #00acdc;
+    background: #fff;
+  }
```

wouldn't it do the same?
```suggestion
  &__button {
    display: flex;
    justify-content: center;
    align-items: center;
    height: 40px;

    color: #fff;
    font-weight: 700;
    font-size: 14px;
    text-transform: uppercase;
    text-decoration: none;

    background: #00acdc;
    border-radius: 5px;
    border: 1px solid #00acdc;

    transition: 0.5s;

      &:hover {
        color: #00acdc;
        background: #fff;
      }
  }
```